container-toolkit/index.md

```diff
@@ -23,16 +23,19 @@ Container Device Interface <cdi-support.md>
 docker-specialized.md
 ```
 
-The NVIDIA Container Toolkit enables users to build and run GPU-accelerated containers.
-The toolkit includes a container runtime [library](https://github.com/NVIDIA/libnvidia-container)
-and utilities to automatically configure containers to leverage NVIDIA GPUs.
+The NVIDIA Container Toolkit is a collection of libraries and utilities enabling users to build and run GPU-accelerated containers.
 
-```{image} https://cloud.githubusercontent.com/assets/3028125/12213714/5b208976-b632-11e5-8406-38d379ec46aa.png
-:width: 400
-```
+The Toolkit currently includes:
```

Comment:
nit: We refer to this as its full name when we reference it. 

```suggestion
The Container Toolkit currently includes:
```

Reply:
Consistency is key, I know. The previous sentence starts with "The NVIDIA Container Toolkit" so I thought don't repeat. 

The table of contents (side bar) shows 

![image](https://github.com/user-attachments/assets/d54539f0-767e-45e7-acdb-30a799ff3ad8)

so I thought "The Toolkit" might be an allowable term to use.

I decide to just do "It" here (using the previous sentence as the reference). Commit upcoming.